docker-app/qfieldcloud/core/admin.py

```diff
@@ -1,8 +1,12 @@
+import csv
 import json
 import time
-from typing import Any, Dict
+from collections import namedtuple
+from typing import Any, Dict, Generator
 
+from allauth.account import admin as allauth_account_admin
```

Comment:
```suggestion
from allauth.account.admin import EmailAddressAdmin
```

Unless the imported items are super generic, please prefer importing directly what you need.